apps/frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Center, Link, Stack, Text } from '@chakra-ui/react';
+import { githubRepo, slackInvite } from '../links';
+import { CompatibleWithStravaIcon } from './CompatibleWithStravaIcon';
+
+export const Footer = () => {
+  return (
+    <Center pb="20vh" px="2">
```

Comment:
Adding some padding to ensure the text is displayed above the `MainActionBar` and `BootomBar` 😰